scripts/listapranzo.js

```diff
@@ -85,6 +85,51 @@ module.exports = function (robot) {
     order.users[user.id] = dishes;
   };
 
+  var countMatches = function (str1, str2) {
+    var s1 = str1.split(' ');
+    var s2 = str2.split(' ');
+
+    s1 = s1.map(function (s) { return s.trim().toLowerCase(); });
+    s2 = s2.map(function (s) { return s.trim().toLowerCase(); });
+
+    var match = 0;
+
+    for (i = 0; i < s1.length; i++) {
+      for (j = 0; j < s2.length; j++) {
+        if (s1[i] === s2[j])
```

Comment:
Mmmhh.. questo sistema di matching non mi convince troppo. Se tipo scrivo pasta pesto matcha con pasta pomodoro nel caso pasta al pesto non ci sia. Fare una cosa più fuzzy dove io scrivo delle lettere tipo "ppesto" e lui da una risposta si/no a seconda se trova quelle lettere (case insensitive) in quell'ordine dentro la stringa? (un po' come cercare `.*[Pp].*[Pp].*[Ee].*[Ss].*[Tt].*[Oo].*`)